package.json

```diff
@@ -30,7 +30,10 @@
     "arrow": "amos-ws16/amos-ws16-arrowjs.git#dev",
     "body-parser": "^1.15.2",
     "express": "^4.14.0",
```

Comment:
Ja, `express` wird immer noch verwendet in `lib/index.js` und `express-promise-router` ist nur ein dünner Wrapper um den Standard `express.router()`, der in der Lage ist mit Promises umzugehen.